docker-entrypoint.sh

```diff
@@ -20,7 +20,7 @@ fi
 
 # if 'tiles' is passed as a command to run, generate and publish tiles
 if [ "$1" == "tiles" ]; then
-    sudo -u postgres /var/lib/postgresql/src/generate_tiles.py
+    sudo -E -u postgres /var/lib/postgresql/src/generate_tiles.py
```

Comment:
?

Reply:
`generate_tiles.py` is now using env variables, so we need to pass them to it.